mxcubeweb/core/components/beamline.py

```diff
@@ -55,21 +55,6 @@ def init_signals(self):
             msg = "error connecting to beamline actions hardware object signals"
             logging.getLogger("MX3.HWR").exception(msg)
 
-        try:
```

Comment:
I agree that this is should be removed and is handled through `stateChanged`, I had a quick look and it seems like "shutterStateChanged" are still used in some places in `mxcubecore` but I think we can consider "shutterStateChanged" deprecated by now and should be replaced with `stateChanged` via `AbstractNState`